### DISCUSSION TITLE
[List & Accordion] How to have accordion within a list with images?

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1524] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hello,
I have a list like this:

![image](https://user-images.githubusercontent.com/5399265/84797225-dbf1dd00-aff9-11ea-9239-f86bca9a89e8.png)

```
<div class="ui relaxed divided list">
  <div class="item">
    <i class="large github middle aligned icon"></i>
    <div class="content">
      <a class="header">Semantic-Org/Semantic-UI</a>
      <div class="description">Updated 10 mins ago</div>
    </div>
  </div>
  <div class="item">
    <i class="large github middle aligned icon"></i>
    <div class="content">
      <a class="header">Semantic-Org/Semantic-UI-Docs</a>
      <div class="description">Updated 22 mins ago</div>
    </div>
  </div>
  <div class="item">
    <i class="large github middle aligned icon"></i>
    <div class="content">
      <a class="header">Semantic-Org/Semantic-UI-Meteor</a>
      <div class="description">Updated 34 mins ago</div>
    </div>
  </div>
</div>
```

As seen [here](https://fomantic-ui.com/elements/list.html#list).

How to add accordion without changing list layout?

I mean that I could click on item, the whole row, and have it expanded like this:

![image](https://user-images.githubusercontent.com/5399265/84798220-fa0c0d00-affa-11ea-9e8a-ca00219f4c48.png)

Then clicking again on the expanded row will close it back again.

What is the best way to implement it?

**Comment by lubber-de** [comment, marked as answer]
- add `title` to the `item` divs
- add another `content` div which is supposed to display your accordion data
See https://jsfiddle.net/lubber/1cLv67kj/3/
![itemsaccordion](https://user-images.githubusercontent.com/18379884/84801971-2bd3a280-b000-11ea-9b39-c9bc1426103c.gif)

Additionally use `exclusive: false` to make each row independent 
See https://jsfiddle.net/lubber/1cLv67kj/6/
![itemsaccordionexclusive](https://user-images.githubusercontent.com/18379884/84802453-e82d6880-b000-11ea-8c0d-7fc4027c1f69.gif)

**Comment by smartm0use** [comment]
Thank you so much!
